Gemfile

```diff
@@ -5,7 +5,7 @@ require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundle
 
 gem 'activerecord-virtual_attributes', '~> 1.5'
 gem 'cloudwatchlogger',                '~> 0.2.1'
-gem 'insights-api-common',             '~> 4.0'
+gem 'insights-api-common',             '>= 5.0.2'
```

Comment:
I meant `'~> 5.0', '>= 5.0.2'` to be sure it won't use version 6

Reply:
done